build.gradle

```diff
@@ -89,8 +88,8 @@ spotbugsTest {
 
 jacocoTestReport {
     reports {
-        xml.enabled = true
```

Comment:
`enabled` is deprecated and was replaced by `required`. Simple `xml` does not work. `html` works, because HTML report is generated by default. I suggest to use:
```gradle
jacocoTestReport {
    reports {
        xml.required = true
        html.required = true
    }
}
```